ui/widgets/mouse.js

```diff
@@ -146,7 +146,16 @@ return $.widget( "ui.mouse", {
 
 			// Iframe mouseup check - mouseup occurred in another document
 			} else if ( !event.which ) {
-				return this._mouseUp( event );
+
+				// Support: Safari 8
```

Comment:
I just tested Safari 9 and the bug exists there as well, so this will be updated to `// Support: Safari 8+` but I'm not sure what you think is wrong with this comment.